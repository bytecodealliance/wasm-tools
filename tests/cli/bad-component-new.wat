;; FAIL: component new %

(module
  (import "a" "a" (func))
  (import "a" "a" (func))
)
