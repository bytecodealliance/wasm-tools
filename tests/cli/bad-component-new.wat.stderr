error: failed to encode a component from module

Caused by:
    0: failed to decode world from module
    1: module was not valid
    2: failed to resolve import `a::a`
    3: module requires an import interface named `a`
